reference/checkouts.v3.yml

```diff
@@ -7933,6 +7934,65 @@ paths:
                 meta: {}
       security:
         - X-Auth-Token: []
+  '/stores/{store_hash}/v3/checkouts/{checkout-id}/token':
```

Comment:
standardize `checkout-id` to `checkoutId` throughout.

Reply:
good catch @slsriehl. I had to create a new branch to merge the changes. (DEVDOCS-4491-feedback)